mono-game/src/Actors/Enemy.cs

```diff
@@ -0,0 +1,39 @@
+using Godot;
+
+public class Enemy : Actor
+{
+
+    [Export]
+    private int score = 100;
+    public override void _Ready()
+    {
+        SetPhysicsProcess(false);
+
+        _velocity.x = -speed.x;
+    }
+
+    public override void _PhysicsProcess(float delta)
```

Comment:
missing a call to `base._PhysicsProcess(delta);`, which causes enemies to ignore gravity.

Reply:
Hi, 
Thank for reviewing.
I fixed the defect.